src/Sylius/Bundle/CoreBundle/Resources/config/app/state_machine/sylius_order.yml

```diff
@@ -26,11 +26,6 @@ winzou_state_machine:
                     do: ["@sylius.order_number_assigner", "assignNumber"]
                     args: ["object"]
                     priority: -200
-                sylius_assign_token:
-                    on: ["create"]
-                    do: ["@sylius.unique_id_based_order_token_assigner", "assignTokenValueIfNotSet"]
-                    args: ["object"]
-                    priority: -100
```

Comment:
I would be for leaving it here, as removing it could be a BC break

Reply:
I tried to imagine but I don't see really a case where it can be a break considering other changes. However, I kept the method and class for BC.